sharedcode/localFooter.html

```diff
@@ -1,44 +1,114 @@
-	<div class="wdn-col-full bp640-wdn-col-two-thirds bp960-wdn-col-one-half">
-		<div id="wdn_footer_contact" class="wdn-footer-module">
-			<span role="heading" aria-level="1">Lincoln EITC Coalition<span class="wdn-text-hidden"> Contact Information</span></span>
-			<div class="wdn-grid-set-halves bp960-wdn-grid-set-halves">
-				<div class="wdn-col">
-					<dl class="wdn-contact-info">
-						<dt>
-							<span class="wdn-icon-location" aria-hidden="true"></span><span class="wdn-text-hidden">Address</span>
-						</dt>
-						<dd>
-							<strong>University of Nebraska&ndash;Lincoln</strong><br />
-							1400 R Street<br />
-							Lincoln, NE 68588<br />
-						</dd>
-						<dt>
-							<span class="wdn-icon-phone" aria-hidden="true"></span><span class="wdn-text-hidden">Phone Number</span>
-						</dt>
-						<dd>
-							<a href="tel:4024727211">402-472-7211</a>
-						</dd>
-						<dt>
-							<span class="wdn-icon-mail" aria-hidden="true"></span><span class="wdn-text-hidden">Email address</span>
-						</dt>
-						<dd>
-							<a href="mailto:vita@unl.edu">vita@unl.edu</a>
-						</dd>
-					</dl>
-				</div>
-			</div>
-		</div>
-	</div>
-	<div class="wdn-col-full bp640-wdn-col-one-third bp960-wdn-col-one-half">
-		<div id="wdn_footer_related" class="wdn-footer-module">
-			<span role="heading" aria-level="1">Related Links</span>
-			<ul class="wdn-related-links-v1">
-				<li><a href="https://events.unl.edu/">UNL Events Calendar</a></li>
-				<li><a href="https://ucomm.unl.edu/">University Communications</a></li>
-				<li><a href="https://www.unl.edu/ucomm/chancllr/">Office of the Chancellor</a></li>
-            </ul>
-		</div>
-	</div>
-</div>
+<nav id="dcf-footer-group-1" role="navigation" aria-labelledby="dcf-footer-group-1-heading">
+	<h2 id="dcf-footer-group-1-heading">Lincoln EITC Coalition</h2>
+	<dl>
+		<dt class="dcf-sr-only">Address</dt>
+		<dd class="dcf-d-inline-block dcf-relative dcf-pl-6">
+			<svg class="dcf-icon dcf-icon-hang dcf-h-4 dcf-w-4 dcf-fill-current" aria-hidden="true" focusable="false"
+			 fill-opacity=".5" height="16" width="16" viewBox="0 0 48 48">
+				<path d="M24 0A16 16 0 0 0 8 16c0 8.49 14.55 30.61 15.17 31.55a1 1 0 0 0 1.67 0C25.45 46.61 40 24.49 40 16A16 16 0 0 0 24 0zm0 23a7 7 0 1 1 7-7 7 7 0 0 1-7 7z" />
+			</svg>
+			<address itemscope="" itemtype="http://schema.org/PostalAddress" translate="no">
+				<a href="https://maps.unl.edu/WAB">
+					<span><span itemprop="name">University of Nebraska–Lincoln</span><br></span>
+					<span itemprop="streetAddress">1400 R Street</span><br>
+					<span itemprop="addressLocality">Lincoln</span>, <abbr title="Nebraska" itemprop="addressRegion">NE</abbr> <span
+					 itemprop="postalCode">68588-0144</span> <abbr class="dcf-d-none" itemprop="addressCountry">US</abbr>
+				</a>
+			</address>
+		</dd>
+		<dt class="dcf-sr-only">Phone</dt>
+		<dd>
+			<a class="dcf-d-inline-block dcf-relative dcf-pl-6" href="tel:+14024729339">
```

Comment:
The phone number specified in the href doesn't match the one below. However, on that same note, is this actually the right phone number haha? 

That number appears to be the University Operator (see https://fleetmanagement.unl.edu/general). I think the right phone number for the Center is 402-472-9638